anonymize.py

```diff
@@ -1,11 +1,49 @@
-from PyPDF2 import PdfReader
+import json
+from presidio_anonymizer import AnonymizerEngine
+from presidio_analyzer import AnalyzerEngine
+from presidio_anonymizer.entities import RecognizerResult, OperatorConfig
 
-reader=PdfReader("meta_projet_final.pdf")
 
+#read txt file
 text=''
-for i in range(len(reader.pages)):
-    page=reader.pages[i]
-    text += page.extract_text()
+with open("info.txt",'r',encoding='utf-8') as file:
```

Comment:
does it work if you remove `encoding="utf-8"` here? encoding is platform-dependent (which means its different for ubuntu and windows), so I cannot test easily on my machine (ubuntu) if it works on windows

Reply:
yes it works , I thought it won't accept french characters without utf-8 but it turns that it does!